codegen/smithy-aws-swift-codegen/src/test/kotlin/software/amazon/smithy/aws/swift/codegen/PresignerGeneratorTests.kt

```diff
@@ -21,7 +21,7 @@ class PresignerGeneratorTests {
             
             extension GetFooInput {
                 public func presign(config: ExampleClientConfigurationProtocol, expiration: Swift.Int64) async throws -> ClientRuntime.SdkHttpRequest? {
-                    let serviceName = "Example"
+                    let serviceName = "example"
```

Comment:
Name of this service changed because it matches the `sdkId` in this test's Smithy def.